src/components/SeeAll.jsx

```diff
@@ -16,9 +16,11 @@ class SeeAll extends Component {
   render() {
     return (
       <div>
-        {this.state.exapnded
-          ? this.props.items
-          : this.props.items.slice(0, this.props.count)}
+        {this.props.items.length > 0
+          ? this.state.exapnded
+            ? this.props.items
+            : this.props.items.slice(0, this.props.count)
+          : `No ${this.props.name} match current filter criteria.`}
```

Comment:
Change the message to `No ${this.props.name} found` as a filter won't always be associated when a `SeeAll` component is used.